movieland_web/src/main/java/com/solovey/movieland/web/controller/GenreController.java

```diff
@@ -9,31 +10,30 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import java.util.List;
+
 import static org.springframework.web.bind.annotation.RequestMethod.GET;
 
 @Controller
-@RequestMapping("/v1/genre")
+@RequestMapping("/genre")
 public class GenreController {
     private final GenreService genreService;
-    private final JsonJacksonConverter jsonConverter;
-
     private final Logger log = LoggerFactory.getLogger(getClass());
 
     @Autowired
-    public GenreController(GenreService movieService, JsonJacksonConverter jsonConverter
+    public GenreController(GenreService movieService
                            ) {
         this.genreService = movieService;
-        this.jsonConverter = jsonConverter;
     }
 
     @RequestMapping(method = GET, produces = "application/json;charset=UTF-8")
     @ResponseBody
-    public String getAllGenres() {
+    public List<Genre> getAllGenres() {
         log.info("Sending request to get all genres");
         long startTime = System.currentTimeMillis();
-        String jsonMovies = jsonConverter.convertGenresListToJson(genreService.getAllGenres());
+        List<Genre> genres = genreService.getAllGenres();
```

Comment:
getAll(), not getAllGenres(), because you're currently in the GenreService

Reply:
done in B-9